script/console

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
```

Comment:
This one includes Aruba::Api to make a quick test easier. It opens a pry console on startup.